test/integration/workarea/storefront/global_e_checkouts_integration_test.rb

```diff
@@ -58,13 +58,12 @@ def test_guest_checkout_clears_user_order_details
         order = Workarea::Order.first
         assert_equal user.id.to_s, order.user_id
 
-        cookies.delete "user_id"
+        travel_to Workarea.config.customer_session_timeout.from_now do
```

Comment:
this replaces the `cookies.delete 'user_id'` call from before. since we can't modify the session in rails integration tests, we'll need to `travel_to` a point in time where the session will expire on its own.